frontend/src/store/modules/v1/worksheet.ts

```diff
@@ -193,10 +194,12 @@ export const useWorkSheetStore = defineStore("worksheet_v1", () => {
     const fullViewWorksheet = getSheetByName(organizer.worksheet, "FULL");
     if (fullViewWorksheet) {
       fullViewWorksheet.starred = organizer.starred;
+      setCache(fullViewWorksheet, "FULL");
     }
     const basicViewWorksheet = getSheetByName(organizer.worksheet, "BASIC");
     if (basicViewWorksheet) {
       basicViewWorksheet.starred = organizer.starred;
+      setCache(basicViewWorksheet, "FULL");
     }
```

Comment:
Is L202 correct? Maybe it's `setCache(basicViewWorksheet, "BASIC").

Reply:
nice catch